### ISSUE TITLE
Using a static generator instead

### ISSUE THREAD

**Issue by leandroferreira** [N-O-D-E/N-O-D-E.net#4]
Would you guys be interest in using a static generator for the posts, such as [Jekyll](https://jekyllrb.com) or [Hexo](https://hexo.io)?

It would basically allow you to:
- keep writing your posts using HTML (or move to Markdown or Stylus)
- auto generate `index.html` and `rss.xml`
- make it easy to make changes to head/footer/etc

Also, if you use Jekyll, you could even write your posts and have it built+hosted using [Github Pages](https://pages.github.com).

If you're interested, i can do the initial setup, migrate the current content and send a PR.

**Comment by JPenuchot** [comment]
Agree, Jekyll is integrated within GitHub Pages, I use it for http://deepspacedreams.com/ and http://jpenuchot.github.io/ using some themes I found online. I do write code but I hate web dev so this saved the day to me as I'm able to use Sublime and Git to edit my websites but still not touching a bit of HTML/CSS/PHP.

**Comment by Serkan-devel** [comment]
<a href="https://github.com/getpelican/pelican/">Pelican</a> would also be a great option.
